src/constants.ts

```diff
@@ -0,0 +1,61 @@
+const constants = Object.freeze({
```

Comment:
I see that `virtualfs` is a dependency for js-encryptedfs.

Well if that's true, you can just use the constants defined inside there.

However I believe `virtualfs` **should not**  be a dependency of `js-encryptedfs` because EFS should just take any fs object in. Why does it need the virtualfs dependency at all?

Reply:
yes `virtualfs` should go into `devDependencies` only for testing purposes. I thought the constants object was a nice little convenience that `virtualfs` provided for defining `mode` in `access` and `open` methods so I stole it from `virtualfs`.